ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -57,6 +57,64 @@ var (
 	podUIDIndexField = "metadata.uid"
 )
 
+// Head pod event list
+const (
+	CreatedHeadPod        = "Created"
```

Comment:
```suggestion
	HeadPodCreated        = "HeadPodCreated"
```

Hi @MadhavJivrajani, we also need to change the string value accordingly.

Reply:
My bad, I overlooked that. Thanks for pointing it out 🙏🏼